google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/JsonToProtoMessageTest.java

```diff
@@ -791,26 +806,30 @@ public void testStructComplex() throws Exception {
                     new BigDecimal("-99999999999999999999999999999.999999999")))
             .setTestNumericStr(
                 BigDecimalByteStringEncoder.encodeToNumericByteString(new BigDecimal("12.4")))
-            .setTestNumericInt(
+            .setTestNumericShort(
```

Comment:
Without out changing v1beta2 code, will this work? Please leave v1beta2 out.

Reply:
The struct is used by both. I can create a new test case but its a bit more ugly.